packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -11,14 +11,17 @@ import { CLIConfigurationService, getRootProjectFolder } from '../util/Extension
 import { ERRORS } from '../service/TranslationKeys';
 import { commandsInfoMap } from '../commandsMap';
 import { assert } from 'console';
+import { window } from 'vscode';
 
 export default abstract class BaseAction {
 	protected readonly translationService: VSTranslationService;
+	protected isFileSelected: boolean | undefined;
```

Comment:
Can we use `protected isFileSelected?: boolean;` notation which seems equivalent and more compact, or is there a difference? I would say the same for `activeFile` property.
Or for `isFileSelected' we could set it initially to `false` so it will always be a boolean.